### PR TITLE
feat(route): add 東網產經

### LINKED ISSUE #11822: money18港股財經新聞
### 类型

金融

### 网站地址

https://money18.on.cc/finnews/news_breaking.html?section=exp

### 网站描述

港股財經新聞

### 需要生成什么内容？

財經新聞

### 额外描述

謝謝!

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved issue

Close #11822

## 完整路由地址 / Example for the proposed route(s)

```routes
/oncc/money18/exp
/oncc/money18/fov
/oncc/money18/industry
/oncc/money18/int
/oncc/money18/recagent
/oncc/money18/ntlgroup
/oncc/money18/pro
/oncc/money18/weainvest
/oncc/money18/ipo
/oncc/money18/tech
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/oncc/money18.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    let items = [];
+
+    if (id === 'ipo') {
+        items = response.data.articles.slice(0, limit).map((item) => ({
+            title: item.title,
+            author: item.authorname,
+            link: `${rootUrl}/finnews/content/${id}/${item.articleId}.html`,
+            description: art(path.join(__dirname, 'templates/money18.art'), {
+                images: item.hasHdPhoto ? [item.hdEnlargeThumbnail] : undefined,
```
Comment (TonyRL) [inline lib/v2/oncc/money18.js:43]: Please add `https://hk.on.cc/hk/bkn` to each img url string

Comment (nczitzk) [reply lib/v2/oncc/money18.js:43]: 👌

File: docs/traditional-media.md [hunk 2]
```diff
@@ -941,6 +943,16 @@ IT・科学 tech_science
 
 </Route>
 
+### 產經
```
Comment (TonyRL) [inline docs/traditional-media.md:946]: I think https://hk.on.cc/hk/finance/index.html is `產經` instead of https://money18.on.cc/index.html

Comment (nczitzk) [reply docs/traditional-media.md:946]: I will move it to `Money18` 😊

**Pull request merged**